2021-11-16 16:24:35 error: uncaughtException: connect ECONNREFUSED 127.0.0.1:27017
MongooseServerSelectionError: connect ECONNREFUSED 127.0.0.1:27017
    at NativeConnection.Connection.openUri (C:\Users\USER\Desktop\Tools\mediaplus\server\node_modules\mongoose\lib\connection.js:846:32)
    at C:\Users\USER\Desktop\Tools\mediaplus\server\node_modules\mongoose\lib\index.js:351:10
    at C:\Users\USER\Desktop\Tools\mediaplus\server\node_modules\mongoose\lib\helpers\promiseOrCallback.js:32:5
    at new Promise (<anonymous>)
    at promiseOrCallback (C:\Users\USER\Desktop\Tools\mediaplus\server\node_modules\mongoose\lib\helpers\promiseOrCallback.js:31:10)
    at Mongoose._promiseOrCallback (C:\Users\USER\Desktop\Tools\mediaplus\server\node_modules\mongoose\lib\index.js:1149:10)
    at Mongoose.connect (C:\Users\USER\Desktop\Tools\mediaplus\server\node_modules\mongoose\lib\index.js:350:20)
    at App.connectToDatabase (C:\Users\USER\Desktop\Tools\mediaplus\server\dist\app.js:45:32)
    at new App (C:\Users\USER\Desktop\Tools\mediaplus\server\dist\app.js:24:14)
    at Object.<anonymous> (C:\Users\USER\Desktop\Tools\mediaplus\server\dist\server.js:13:13)
    at Module._compile (node:internal/modules/cjs/loader:1101:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1153:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)
    at node:internal/main/run_main_module:17:47
